recursion/ex2_fromOnetoEight.py

```diff
@@ -0,0 +1,19 @@
+# ex 2
+
+def naturalNumbers_backward(n: int):
+    '''
+    A recursive function prints first n natural numbers
+    in reverse order
+    '''
+    if n < 1:
+        print("Error: please choose a natural number.")
+    elif n == 1:
+        print(n, end = ", ")
+    else:
+        naturalNumbers_backward(n-1)
```

Comment:
Well, other types of writing are not accepted?

Reply:
It's a convention. You might work in a company that decides that writing functions is done in CamelCase for an example. The default convention is snake_case so for now we will stick to it So that our code will be clean and coherent.